lib/Sema/CSSimplify.cpp

```diff
@@ -2161,22 +2161,37 @@ static ConstraintFix *fixPropertyWrapperFailure(
     }
   };
 
+  // Local function to substitute into the
+  auto substMemberType = [&](VarDecl *var, Type type) -> Type {
+    if (!baseTy)
```

Comment:
I think this is more of an assert then a check since there should away be a base type where looked up original property on...

Reply:
One can use property wrappers with (e.g.) global properties, in which case there is no base type and therefore shouldn't be any substitution.